samples/consul-ui/decorators.js.erb

```diff
@@ -36,6 +36,13 @@ function usefullLinksGenerator(instance, serviceName, node_meta_info) {
     return top;
 }
 
+/**
+ * instanceNameDecorator is called to resolve instance name.
+ */
+function instanceNameDecorator(instance, nodemeta) {
```

Comment:
I would rather call it: `createNodeDisplayElement(nodeName, nodeMeta)`

and return by default : `document.createTextNode(nodeName)` => So it can be used to display specific icons for instance